__tests__/utils.test.js

```diff
@@ -182,6 +171,7 @@ describe("GET /api/articles/:article_id", () => {
         expect(data.hasOwnProperty("created_at")).toBe(true);
         expect(data.hasOwnProperty("votes")).toBe(true);
         expect(data.hasOwnProperty("article_img_url")).toBe(true);
+        expect(data.hasOwnProperty("comment_count")).toBe(true);
```

Comment:
Please don't amend previous tests. This should be a new test where we would only need to check for the comment_count property